website/events/serializers.py

```
@@ -1,6 +1,10 @@
 from rest_framework import serializers
-from .models import Event
-import uuid
+from .models import Event, Resource
+
+class ResourceSerializer(serializers.ModelSerializer):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)